### DISCUSSION TITLE
HvTable sort - Only sort based on ascending or descending.

### DISCUSSION THREAD

**Discussion by sarika-jamdade** [lumada-design/hv-uikit-react#3169] [Q&A] [answered]
Hello
 
 We started using hitachivantara UI components for the development of SMS Tool project. We are using 
@hitachivantara/uikit-react-core package. Have added HvTable in our project and need a clarity on HvTable components sort option. 
By default, data is sorted based on Ascending and Descending order. But the sort result giving three combinations
1. Sorted in Ascending order.
2. Sorted in Descending order.
3. Unsorted - This displays the data in the same order as it was entered in the database.

 Is there any way to only activate the two options of sort that is either Ascending or Descending and not to use 3rd option of sort

**Comment by zettca** [comment, marked as answer]
Hey @sarika-jamdade 👋 

Sorry for the delay. I think you're looking for the `disableSortRemove` configuration.

Closing this as resolved
